### DISCUSSION TITLE
openwrt，router wdr7500v3，usb mic，rtsp audio to this server in docker ，the voice is continue from time to time

### DISCUSSION THREAD

**Discussion by yunhai20082008** [bluenviron/mediamtx#447] [Questions]
shell in openwrt:

tiangolo/nginx-rtmp:
can be executed,the vioce is normal:
ffmpeg -f alsa -acodec pcm_s16le -ac 1 -ar 44100 -i hw:0,0 -f rtsp "rtsp://192.168.20.1:8554/live"

aler9/rtsp-simple-server:
both cant be executed,logs ERR: unsupported audio codec 1:
ffmpeg -f alsa -acodec pcm_s16le -ac 1 -ar 44100 -i hw:0,0 -f flv "rtmp://192.168.20.1:1935/live"

can be executed,the voice is continue from time to time,and shell has many like [alsa @ 0x773b7890] ALSA buffer xrun. speed=0.997x:
ffmpeg -f alsa -acodec pcm_s16le -ac 1 -ar 44100 -i hw:0,0 -f rtsp "rtsp://192.168.20.1:1935/live"

how to fix this

**Comment by aler9** [comment]
Hello, the problem is that at the moment the RTMP server doesn't support the PCM codec. This feature may be added in the future.

**Reply by yunhai20082008** [reply]
what about the rtsp server? that command can be executed,but the vioce is continue from time to time

[a comment by an automated account is not shown]
